templates/data-store-pv.yaml

```diff
@@ -12,37 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Keep track of the root . inside control structures
-{{- $root := . }}
+# Keep track of the 'hosting' tag inside control structures, as '.' is overridden and '.Values' is inaccessible
+{{- $hosting := .Values.hosting }}
```

Comment:
this variable doesn't seem to serve any purpose, if it is not to be mutated use the .Values notation.

Reply:
The range control structure overrides the value of `.` to whatever range item you are operating on, making `.Values` point to (in this case) `global.persistence.dataStore.palisade-data-store.Values`

Reply:
Changed to $, t.i.l.